modules/local/interpro_summary.nf

```diff
@@ -4,15 +4,17 @@ process INTERPRO_SUMMARY {
     label 'process_single'
 
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/csvtk:0.31.0--h9ee0642_0':
-        'biocontainers/csvtk:0.31.0--h9ee0642_0' }"
+        'oras://community.wave.seqera.io/library/csvtk_tabix_pip_biopython:e6e033af2a05a562':
+        'community.wave.seqera.io/library/csvtk_tabix_pip_biopython:7eabdb397e7420a3' }"
 
     input:
     tuple val(meta), path(interproscan_tsv)
 
+    // The gzi are optional, it's possible that the summaries are empty and in that case the .gzi are not created
```

Comment:
Forgive me, I am missing how the empty `.gzi`s are *not* emitted / filtered out?

Reply:
Fixed here -> https://github.com/EBI-Metagenomics/assembly-analysis-pipeline/pull/3/commits/f63541ddc9e7c5fbf576369552cf61cb7f967ee2#diff-d6e257baadf8e3281a8ac521113c49d301c2eebe2af42aec6821a8462674fc7cR39